protobufs/funding/funding.proto

```diff
@@ -87,12 +87,12 @@ message FundingUpdatePayload {
 
 message FundingResponse {
   document.ResponseHeader header = 1;
-  FundingData data = 2;
+  FundingResponseData data = 2;
 }
 
 message FundingListResponse {
   document.ResponseHeader header = 1;
-  repeated FundingData list = 2;
+  repeated FundingResponseData list = 2;
```

Comment:
`FundingListResponse` that has a property `list` sounds redundant to me. Can we follow the same pattern and keep the name as `data`.